### DISCUSSION TITLE
Seed creation in peak coordinates

### DISCUSSION THREAD

**Discussion by pedrohenrique1992** [simnibs/simnibs#184] [9. Results and analysis]
Hello!

I would like to know how to locate the field magnitude peak in the surface file in fsaverage space and from that location generate a ROI centered on the peak coordinate or a ROI that contains all voxels with the peak value to be inserted as a seed under analysis functional connectivity on CONN (as a nifti file in fsaverage space).

Thanks

**Comment by oulap** [comment]
You can have a look at these two matlab scripts.

https://github.com/simnibs/simnibs/blob/master/simnibs/examples/analysis/example_get_data_from_fs_files.m

https://github.com/simnibs/simnibs/blob/master/simnibs/matlab_tools/mesh_get_fieldpeaks_and_focality.m

The first script shows how to read in a freesurfer surface, and the second one is a function to get the xyz-coordinate of the e-field max. The script calls the function for extracting these values.
